tests/dashboard/components/file/test_views.py

```diff
@@ -0,0 +1,405 @@
+import json
+import uuid
+from unittest import mock
+
+import elasticSearchFunctions
+import pytest
+from components import helpers
+from django.urls import reverse
+from main import models
+
+
+@pytest.fixture
+def dashboard_uuid(db):
+    helpers.set_setting("dashboard_uuid", str(uuid.uuid4()))
+
+
+@pytest.fixture
+def transfer(db):
+    return models.Transfer.objects.create()
+
+
+@pytest.fixture
+def sip(db):
+    return models.SIP.objects.create()
+
+
+@pytest.fixture
+def file(db):
+    return models.File.objects.create()
+
+
+def test_file_details(admin_client, dashboard_uuid, file, mocker):
+    mocker.patch("elasticSearchFunctions.get_client")
+    mocker.patch(
+        "elasticSearchFunctions.get_transfer_file_info",
+        return_value={
+            "filename": "LICENSE",
+            "fileuuid": str(file.uuid),
+            "sipuuid": "fbca5a00-9b05-4b3d-b270-e43a72c2c099",
+            "status": "backlog",
+            "size": 0.032919883728027344,
+            "tags": [],
+            "bulk_extractor_reports": [],
+            "format": [
+                {"puid": "x-fmt/111", "format": "Generic TXT", "group": "Text (Plain)"}
+            ],
+            "pending_deletion": False,
+        },
+    )
+    response = admin_client.get(reverse("file:file_details", args=[file.uuid]))
+    assert response.status_code == 200
+
+    content = json.loads(response.content.decode("utf8"))
+    assert content["id"] == str(file.uuid)
+    assert content["title"] == "LICENSE"
+
+
+@pytest.mark.parametrize(
+    "mock_file_info, status_code",
+    [
+        ("get_transfer_file_info returned no exact results", 404),
+        ("unknown error", 500),
+    ],
+    ids=["no exact results", "unknown error"],
+)
+def test_file_details_handles_exceptions(
+    admin_client, dashboard_uuid, file, mocker, mock_file_info, status_code
```

Comment:
Let's keep `parametrize` or `patch` decorators as the initial parameters of the test functions. Fixtures can come after.

```suggestion
    mock_file_info, status_code, admin_client, dashboard_uuid, file, mocker
```